plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -57,6 +57,10 @@ jQuery( function ( $ ) {
 		} );
 	} );
 
+	$( document.body ).on( 'woocommerce_attributes_saved', function () {
+		$( '#product_attributes_actions' ).removeClass( 'hidden' );
```

Comment:
The UI allows the attributes to be saved when not filled out. So, a save can result in zero attributes. We should check the number of attributes before showing these actions.